tests/unit_tests/test_ustar_cp/test_launch.py

```diff
@@ -198,3 +208,235 @@ def test_missing_keywords(setup_test_environment, matlab_engine):
 
         # Add the current line to the partial data  for the next test
         partial_sample_data += ",".join(line) + "\n"
+
+
+loadData_test_cases = ['2005', '2006']
+@pytest.mark.parametrize('year', loadData_test_cases)
+def test_loadData(year, setup_folders, matlab_engine):
```

Comment:
I feel `test_loadData` looks more like regression test as it compares with the original data

Reply:
It solely checks that the data is loaded/read correctly so is only a single thing